src/components/PerformanceComparison/PerformanceComparisonData.ts

```diff
@@ -60,18 +60,18 @@ export const PerformanceComparisonData: Record<
     table: [
       {
         recordSize: "10 MB",
-        readSpeed: "+52%",
-        writeSpeed: "+217%",
+        readSpeed: "+10%",
+        writeSpeed: "+361%",
       },
       {
         recordSize: "1 MB",
-        readSpeed: "+170%",
-        writeSpeed: "+333%",
+        readSpeed: "+677%",
```

Comment:
the blog is at +79%, or is it for 100 KB?

Reply:
yes you're right. Fixed